server/models/Request.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user: { 
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "user",
+    required: true
+  },
+  sitterId: {
```

Comment:
Won't we need to add a reference to the `User` model here as well? :)

Reply:
Yes I think so, and that is what I did in line 6 `ref: "user"`. I used lower case `"user"` and not capitalized `"User"`  because I was getting this error when I make get requests to the /requests endpoint with a capitalized `"User"`:

![schema mismatch](https://user-images.githubusercontent.com/53505772/133533618-52c72fe3-15ae-4d12-ad10-f8d10c452ee7.PNG)

 I initially thought that it was because a lower case `"user"` was used to define the model, as in `User = mongoose.model("user", userSchema)` [check it out here](https://github.com/hatchways/team-brioche/blob/052672f7c5d7a5d8467c72524d6a2727820c48d6/server/models/User.js) and a mismatch occurs if I use a capitalized user `ref:"User"`. I now realize that the error was coming from my use of the populate method in the request controller which was implemented this way `const request = Request.find(...).populate("user")` with a lower case, it should be `populate("User")` . I will update both files now. thanks for the heads up 👍

Reply:
Good job on debugging this error 👍🏽